homework-19/script.js

```diff
@@ -0,0 +1,82 @@
+class ApartamentBuilding {
+    constructor(street, number, floorsNumber, apartamentsNumber, dwellersNumber) {
+        this.adress = {sreet: street, number: number};
+        this.floorsNumber = floorsNumber;
+        this.apartamentsNumber = apartamentsNumber;
+        this.apartamentsArray = [0];
+        this.dwellersNumber = dwellersNumber;
+    }
+
+    addApartamentInfo(apartament) {
+        if (apartament.number > this.apartamentsNumber) {
+            return;
+        } else {
+            this.apartamentsArray[apartament.number] = apartament;
+        }
+    }
+
+    getApartament(number) {
+        if (number > this.apartamentsNumber) {
+            return `Такої квартири не існує`;
+        }
+        return this.apartamentsArray[number] ?? `Інфо про цю квартиру ще немає`;
+    }
+}
+
+class Apartament {
+    constructor(floor, number, area, roomsNumber) {
+        this.floor = floor;
+        this.number = number;
+        this.area = area;
+        this.roomsNumber = roomsNumber;
+        this.dwellersArray = [];
+    }
+
+    addDwellerInfo(dweller) {
```

Comment:
```suggestion
    addDweller(dweller) {
```
Ви додаєте екземпляр класу Мешканець, а не інформацію про нього